ras_party/controllers/party_controller.py

```diff
@@ -226,14 +226,14 @@ def change_respondent(payload, tran, session):
     email_address = payload['email_address']
     new_email_address = payload['new_email_address']
 
-    respondent = session.query(Respondent).filter(Respondent.email_address == email_address).one()
+    respondent = _query_respondent_by_email(email_address, session).one()
```

Comment:
Needs to be a .first() else requires a try except.

Reply:
Good spot